components/frontend/src/App.js

```diff
@@ -39,16 +39,9 @@ class App extends Component {
             reportDate.setHours(23, 59, 59);
         }
         this.login_forwardauth();
+        this.initUserSession()
         this.connect_to_nr_measurements_event_source();
-        if (new Date(localStorage.getItem("session_expiration_datetime")) < new Date()) {
-            this.set_user(null)  // The session expired while the user was away
-        }
-        this.setState(
-            {
-                report_uuid: report_uuid, report_date: reportDate, loading: true,
-                user: localStorage.getItem("user"), email: localStorage.getItem("email")
-            },
-            () => this.reload());
+        this.setState({ report_uuid: report_uuid, report_date: reportDate, loading: true }, () => this.reload());
```

Comment:
Nice simplification from before. I guess converting `report_uuid` to camelCase is annoying :o)

Reply:
Yeah, that's a too big change, it's used everywhere.